source/_integrations/verisure.markdown

```diff
@@ -104,6 +104,19 @@ giid:
   type: string
 {% endconfiguration %}
 
+## 2 Factor Authentication Prerequisite
+
+Verisure added 2FA rules to Verisure My Pages that aren't supported through their third party API integration. If you have 2FA enabled (which is forced as default) You might not be able to use this integration. Here is the suggested way to deactivate 2FA if it's allowed in your region.
+
+You can deactivate 2FA for your admin account and use that for Home Assistant but this isn't recommended. The steps below sets up a specific Home Assistant user and gives it restricted access.
+
+1. Log in to Verisure My Pages as your admin user and create a new admin user for Home Assistant.
+2. Log in in as your newly created Home Assistant user, you'll be prompted to set up 2FA, do that and then log out. This will make sure the options below are available.
+3. Log in as the Home Assistant user, browse to Account and subscription -> Account -> Login Credentials -> Disable 2FA.<div class='note warning'>This will only be available if the user is admin and has logged in once with 2FA, logged out and in again.</div>
+4. Log in as your administrator again and change the Home Assistant user to a restricted user.
+5. Change Home Assistant Verisure config to the new user credentials in Home Assistant.
+6. Restart Home Assistant.
```

Comment:
```suggestion
1. Log in to Verisure My Pages as your admin user and create a new admin user for Home Assistant.
2. Log in as your newly created Home Assistant user, you'll be prompted to set up 2FA, do that and then log out. This will make sure the options below are available.
3. Log in as the Home Assistant user, browse to Account and subscription -> Account -> Login Credentials -> Disable 2FA.<div class='note warning'>This will only be available if the user is admin and has logged in once with 2FA, logged out and in again.</div>
4. Log in as your administrator again and change the Home Assistant user to a restricted user.
5. Change Home Assistant Verisure config to the new user credentials in Home Assistant.
6. Restart Home Assistant.
```